### ISSUE TITLE
Support for BlockDevice volume subtype with `podman play kube`

### ISSUE THREAD

**Issue by tupyy** [containers/podman#13951]
**Is this a BUG REPORT or FEATURE REQUEST? (leave only one on its own line)**

/kind feature

**Description**
Support of the `BlockDevice` volume subtypes in `hostPath` type.

**Comment by tupyy** [comment]
Hello,

Our use case is to access devices in the container when they are created with `podman play kube` and we would like to use `Container Device Interface`. 
I was thinking of adding a new annotation for the device name in the `yaml` file like:
```
device.podman.io/vendor=vendor_name
device.podman.io/name=device_name
```

What do you think is the best approach here? 

Thanks

**Comment by vrothberg** [comment]
Removing the "Good First Issue" label as the implementation/design is unclear at the moment.

@containers/podman-maintainers opinions/ideas?

**Comment by tupyy** [comment]
Solved with PR #14266
